vnext/Shared/OInstance.cpp

```diff
@@ -544,7 +547,8 @@ void InstanceImpl::loadBundleInternal(std::string &&jsBundleRelativePath, bool s
       }
 
 #else
-      std::string bundlePath = m_devSettings->bundleRootPath + jsBundleRelativePath + ".bundle";
+      std::string bundlePath =
+          (fs::path(m_devSettings->bundleRootPath) / (jsBundleRelativePath + ".bundle")).u8string();
```

Comment:
>/ [](start = 51, length = 1)

Is this an operator overload for FS paths?

Reply:
Yes, this is how the std::filesystem concatenates paths. 

---
In reply to: [433453526](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433453526) [](ancestors = 433453526)